### ISSUE TITLE
GitHub Actions are not running on PR submitted from a fork

### ISSUE THREAD

**Issue by skel35** [toggl-open-source/toggldesktop#3971]
### 💻 Environment
Platform: macOS/Windows/Linux

### 📒 Description
GitHub Actions are not running on PR submitted from a fork.
E.g. #3970

**Comment by MartinBriza** [comment]
Yeah, I guess this is a security feature - we'll need a special set of actions to be ran on PRs while making extra sure there are no credentials to be leaked

**Comment by skel35** [comment]
Oh, you're totally right, I've reported an issue without thinking about it.

Now that I've googled it, GA can be run on a fork in the forked repo itself if the owner of that forked repo enables Actions. Otherwise, it is a feature indeed.

This issue is invalid then.

**Comment by MartinBriza** [comment]
I think it's possible to run workflows on PRs though, but I'm not sure about it.

**Comment by MartinBriza** [comment]
There's a short blurb about it in the docs: https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories
So it should be possible, but requires special care, if I understand it right. I'll reopen the issue so we keep it as a future possibility to improve
